src/components/Location.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+// TODO: Location styling/icon that Siraj wants
+const LocationContainer = styled.button`
+  width: 10px;
+  height: 10px;
+  border-radius: 100%;
+  background: red;
+  transform: translate(-50%, -50%);
+`
+
+/**
+ * Component for a location displayed on the map.
+ * @param {function} onClick - The handler called when this location is clicked
+ */
+const Location = ({ onClick }) => <LocationContainer onClick={onClick} />
```

Comment:
Also, a styled-components tip here: you don't have to explicitly create a new `Location` component. This is the same as LocationContainer because buttons also have an onClick prop. So you could remove this line and rename the above to Location.

Reply:
I believe you can move the documentation to just above line 6, by the way.